examples/jsm/lines/Line2.d.ts

```diff
@@ -4,6 +4,9 @@ import { LineMaterial } from './LineMaterial';
 
 export class Line2 extends LineSegments2 {
 
+	geometry?: LineGeometry;
+	material?: LineMaterial;
```

Comment:
This PR produces now linter warnings:

> Type 'undefined' is not assignable to type 'Geometry | BufferGeometry'.

Same for the material property definition:

> Type 'undefined' is not assignable to type 'Material | Material[]'.

@giulioz Do you mind checking this?

Reply:
Will check ASAP